spec/forms/teacher_interface/delete_upload_form_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe TeacherInterface::DeleteUploadForm, type: :model do
+  describe "#validations" do
+    subject(:form) { described_class.new(confirm:, upload:) }
+
+    context "when the fields are blank" do
+      let!(:confirm) { "" }
```

Comment:
```suggestion
      let(:confirm) { "" }
```

I don't think we need any of the `confirm` definitions to be non-lazy as they don't have any side-effects (creating objects, etc).